File: Python/gob.py
import sys
import time

print "Gob's program? y/n"
op = sys.stdin.readline();

if op.strip() != 'y':
	sys.exit(0)
	

while True:
	print "Penus",
	sys.stdout.flush()
	time.sleep(0.1)

File: Rust/gob.rs
use std::{io, time};

fn main() {
    println!("Gob's program: y/n?");
    let x = io::stdin().read_line().unwrap();
    if x.as_slice() == "y\n" {
        loop {
            print!("Penus ");
            std::io::stdio::flush();
            std::io::timer::sleep(time::Duration::milliseconds(100))
        }
    }

}
